packages/service-clients/tinylicious-client/src/index.ts

```diff
@@ -23,3 +23,5 @@ export {
 	type TinyliciousUser,
 } from "./interfaces.js";
 export { TinyliciousClient } from "./TinyliciousClient.js";
+
+export type { CompatibilityMode } from "@fluidframework/fluid-static";
```

Comment:
Can we add a brief comment to each of the cross-package re-exports explaining why they're useful? This will make it easier for devs to reason about if/when it would be appropriate to remove them in the future.

Reply:
Added in latest